__tests__/html/hooks.useCreateActivityStatusRenderer.html

```diff
@@ -51,6 +51,7 @@
         );
 
         await pageConditions.webChatRendered();
+        await pageConditions.uiConnected();
```

Comment:
Without this, we would send the incoming activity before `connectionStatus$` turn into `ONLINE`.